.github/workflows/ci.yml

```diff
@@ -64,18 +56,10 @@ jobs:
             build-type: Release
             build-dir: .build
             cxx-flags: ${{ null }}
-            skeleton: ${{ false }}
+            skeleton: ${{ true }}
```

Comment:
Here and in all other places: could you please rename `linux-gcc-release-ootree` to  `linux-gcc-release-ootree-skeleton` to follow common naming pattern?

Reply:
Done!